packages/stark-ui/assets/styles/_header.scss

```diff
@@ -181,3 +160,16 @@
     }
   }
 }
+
+@media #{$mobile-only-query} {
```

Comment:
Also for these styles: the same as for my previous remark, we should not add this rules in `stark-ui`, they should be in the showcase instead...